client/src/helpers/apiClient.js

```diff
@@ -1,20 +1,26 @@
 import axios from 'axios';
 const apiUrl = process.env.REACT_APP_API_URI || 'http://localhost:3001';
 
+ 
 const apiClient = {
     getCategories: () => {
-        return axios.get(`${apiUrl}/api/categories`);
-    },
+             return axios.get(`${apiUrl}/api/categories`);              
+     },
     getArticles: () => {
         return axios.get(`${apiUrl}/api/articles`);
     },
     getSingleArticle: (articleId) => {
 
         return axios.get(`${apiUrl}/api/articles/${articleId}`);
     },
-    getArticlesByCategoryId: (categoryId) => {
+    getArticlesByCategoryId: (categoryId,language) => {
 
-        return axios.get(`${apiUrl}/api/categories/${categoryId}`);
+        if(language=="En"){
```

Comment:
this is not the right logic - the if and else are the same now. It should be one call to axios.get - check if language is NOT provided, then default to English